src/components/CoverImage.jsx

```diff
@@ -0,0 +1,7 @@
+
+
+export const CoverImage = () => {
+    return (
+        <div>CoverImage</div>
```

Comment:
The function will pass a url in this case.
E.g:
`export const CoverImage = ({ imageUrl }) => {
    return (
        <div>
            <img src={imageUrl} alt="album cover image" />
        </div>
    )
}`